bundles/org.openhab.ui/README.md

```diff
@@ -0,0 +1,3 @@
+# Default UI
```

Comment:
We will definitely have to come up with some proper naming.
"Default" sounds too technical to me, I'd actually prefer to just call it the "openHAB UI". Or maybe to better distinguish it from the apps, the "Web UI". Or maybe the "Standard UI"...

Also, we have to decide whether this README gets consumed by the documentation build or if we have all infos about this UI in the docs repo (as it is spread across many sections, this is probably the better choice).

Reply:
Agreed that's why I simply named the package "org.openhab.ui" instead of "org.openhab.ui.default" or something like this (it wouldn't accept "default" in a package name anyway). I'll come up with a proper README later.

> Also, we have to decide whether this README gets consumed by the documentation build or if we have all infos about this UI in the docs repo (as it is spread across many sections, this is probably the better choice).

Since this would be the go-to UI I think would be indeed be described in the docs as part of openHAB instead of a specific feature.